schemas/log.yml

```diff
@@ -43,3 +43,28 @@
       short: Name of the logger.
       description: >
         The name of the logger inside an application. This is usually the name of the class which initialized the logger, or can be a custom name. 
+
+    - name: origin.file
```

Comment:
Let's reuse the `file` object here: https://github.com/elastic/ecs/blob/master/schemas/file.yml#L15

So this becomes `log.origin.file.path` I guess?

Reply:
Good call! It would be `log.origin.file.name` though. Should we then also do `log.origin.file.line` instead of `log.origin.line`?

Reply:
That would make sense I guess. I wonder if we "must" extend `file` for this or we just use it here? I'm good with just adding it here but I'm sure at @webmat will have an opinion on this.

Reply:
I'm fine with directly adding 2-3 specific fields for now 👍 

We're going for something like this for `package.*` as well

Reply:
Note: Using the `file.*` object here would require a change to ECS, since it is not currently defined as a reusable (nestable) object.  I'm not saying this is wrong, as the idea has come up before to capture events like file-copy events, but we should consider this as we work through this PR, addressing where else `file.*` could be nested.  For now, I would suggest we don't re-use the `file.*` object here.

Reply:
It's not using the full `file` object here but re-uses the same structure. Coming up with slightly different names just not to re-use the `file` object structure doesn't feel quite right. But I think I don't quite understand what you have in mind here. Which names would you suggest?